impeller/compiler/compiler.cc

```diff
@@ -20,6 +20,24 @@
 namespace impeller {
 namespace compiler {
 
+static std::optional<spv::ExecutionModel> SourceTypeToExecutionModel(
+    const SourceType& type) {
```

Comment:
nit: `SourceType` is just an enum, so we'd usually pass it by value.
```suggestion
    SourceType type) {
```